gc/startup/omrgcstartup.cpp

```diff
@@ -335,3 +344,9 @@ OMR_GC_ShutdownHeap(OMR_VM *omrVM)
 	return OMR_ERROR_NONE;
 }
 
+omr_error_t
+OMR_GC_ShutdownHeapAndCollector(OMR_VM* omrVM) {
+	OMR_GC_ShutdownCollector(omrVM);
+	OMR_GC_ShutdownHeap(omrVM);
+	return OMR_ERROR_NONE;
+}
```

Comment:
Can you add a newline at the end of the file please?

Reply:
Fixed.